### PR TITLE
Fixed "tokudb.parts" MTR test suite.

### PR DESCRIPTION
**Pull request by percona-ysorokin**
Fixed invalid DEBUG_SYNC point name in "partition_debug_sync_tokudb.test" as it has
been renamed from "before_rename_partitions" to "before_handle_alter_part_end".

Fixed invalid "--source suite/parts/inc/partition_max_<xxx>.inc" directives, as these
files have been moved to "suite/max_parts/inc/partition_max_<xxx>.inc".

Re-recorded test results files as widly used "SET sql_mode =''" in 5.7 generates warnings.

Please notice, that some "tokudb.parts" tests require max number of OS's open files to be
no less than 32768.
When MTR is invoked in parallel mode, say, with "--parallel=4", then this value must
be increased even more to 32768 \* 4 = 131072.

To configure these values on CentOS 7 use the following instructions:
https://naveensnayak.wordpress.com/2015/09/17/increasing-file-descriptors-and-open-files-limit-centos-7/

To sum up:

In /etc/sysctl.conf
fs.file-max = 131072

In /etc/security/limits.conf
- soft nproc 32768
- hard nproc 32768
- soft nofile 32768
- hard nofile 32768

Then invoke
sysctl -p

### CODE REVIEW

**Comment by percona-ysorokin** [comment]
Jenkins build link
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/84/
